config/eslintrc_react.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
         'react/no-is-mounted': 2,
         'react/no-multi-comp': 0, // Enable to define a multiple component to a single file.
         'react/no-render-return-value': 2,
+        'react/no-redundant-should-component-update': 0, // we'll reconsider this option when we begin to use `PureComponent`.
```

Comment:
I think we should enable this. React v15.3 introduced `PureComponent`. There is no blocker to use it. We can use it anytime.